src/signals/incident/components/IncidentForm/index.test.js

```diff
@@ -257,9 +259,7 @@ describe('<IncidentForm />', () => {
         next,
       });
 
-      defer(() => {
-        expect(next).not.toHaveBeenCalled();
-      });
+      expect(next).not.toHaveBeenCalled();
```

Comment:
Same here; assert that `next` has not been called before any changes to have been applied.